cirq-google/cirq_google/engine/engine_processor_test.py

```diff
@@ -301,6 +301,23 @@ def test_get_missing_device():
         _ = processor.get_device()
 
 
+def test_get_sampler_initializes_default_device_configuration() -> None:
+    processor = cg.EngineProcessor(
+        'a',
+        'p',
+        EngineContext(),
+        _processor=quantum.QuantumProcessor(
```

Comment:
Can you also add a test for when no processor is provided to the constructor? (you can rely on a fake `EngineContext`)